common.gradle

```diff
@@ -1,4 +1,7 @@
-project.ext.hasBackend = false
+if (project.hasProperty("hasBackend"))
```

Comment:
What is the purpose of this change? Why is it necessary?

Reply:
@liaxim This demo is for monoscopic mode only. How could we configure our project to build with monoscopic backend only?

Our try was done at commit 9ab5547d165d69d6e70a6978314fe0daef5bb530